addOns/spider/src/main/java/org/zaproxy/addon/spider/parser/SpiderHttpHeaderParser.java

```diff
@@ -34,14 +38,12 @@ public boolean parseResource(ParseContext ctx) {
         HttpMessage message = ctx.getHttpMessage();
 
         // Content-location header
-        // TODO replace when next core version available HttpHeader.CONTENT_LOCATION
-        String location = message.getResponseHeader().getHeader("Content-Location");
+        String location = message.getResponseHeader().getHeader(HttpHeader.CONTENT_LOCATION);
```

Comment:
I'd use the ones from commonlib instead (`HttpFieldsNames`).

Reply:
Okay